### ISSUE TITLE
C/C++ Theme "Dark (Visual Studio C/C++)" theme doesn't match the latest VS C++ colors

### ISSUE THREAD

**Issue by sean-mcmanus** [microsoft/vscode-cpptools#6108]
See the screenshot comparison:

![image](https://user-images.githubusercontent.com/19859882/92810450-68e6d100-f372-11ea-8822-5ee5bfb180d7.png)

**Comment by xiaoma20082008** [comment]
@sean-mcmanus 
Hello, I've got a new color error in the comment, see bellow:
![image](https://user-images.githubusercontent.com/10076393/98810607-71cf4e00-245a-11eb-9733-ae19a0146e3c.png)

vscode version: 
```
Version: 1.51.0
Commit: fcac248b077b55bae4ba5bab613fd6e9156c2f0c
Date: 2020-11-05T18:14:40.758Z
Electron: 9.3.3
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Darwin x64 19.6.0
```

**Comment by Colengms** [comment]
Hi @xiaoma20082008 .  This issue is tracking the need to make arbitrary changes to colors in a theme, and is not related to colorization bugs.  Normally, I'd suggest opening a new issue for this, but I suspect this repro's with the C/C++ Extension disabled, and is by design.

The C/C++ Extension provides semantic colorization (colorization for tokens that require compilation to identify).  VS Code itself still provides the underlying syntactic colorization (all colorization you see when the C/C++ Extension is disabled) using TextMate grammar rules.  Those rules are contributed to VS Code from [the repo here](https://github.com/jeff-hykin/cpp-textmate-grammar).  Though, I believe you are seeing an intended feature related to use of `/**` vs. `/*` and documentation comments (perhaps Doxygen).

**Comment by xiaoma20082008** [comment]
Thanks for your replying  @Colengms 
The dark theme(Visual Studio C/C++) is my default theme and the C/C++ Extension is enabled, but it's also appearing.
Why I commented here is that I checked the issues then found this issue is similar to my case but I'm not sure if other user had open a same issue or a new future also I'm not sure whether it's a bug . 

ps: I tried the [TextMate](https://github.com/jeff-hykin/cpp-textmate-grammar) c/c++ color, and it works wrong too.

**Comment by BlueStaggo** [comment]
This is the reason why I have made an updated version of the theme. Here's a comparison between the current theme (probably based off of VS2017) and my theme (based off of VS2019 16.9):

## Dark

### C / C++ Extension
![image](https://user-images.githubusercontent.com/71090710/125194888-23718980-e24b-11eb-95ae-9b11396a888e.png)

### Recreation of VS2019
![image](https://user-images.githubusercontent.com/71090710/125194908-3c7a3a80-e24b-11eb-9e35-116b591b34bd.png)

## Light

### C / C++ Extension
![image](https://user-images.githubusercontent.com/71090710/125194970-6f243300-e24b-11eb-98c6-897ba5f4bf6e.png)

### Recreation of VS2019
![image](https://user-images.githubusercontent.com/71090710/125194992-895e1100-e24b-11eb-8d57-a1843f8a4490.png)

There is an obvious difference between these themes. If you want to use my version, [get it here](https://gist.github.com/BlueStaggo/b8769f3d4fa66b77e02bebcc1d204277). Feel free to tweak it all you want, as this may not be fully accurate. In fact, you can leave the current theme in and add in my theme alongside it.

**Comment by BlueStaggo** [comment]
Note: I said this theme may not be fully accurate, and that might be true for the pragmas. Just set `meta.preprocessor` to the default foreground colour. That is probably what it looks like in Visual Studio.

**Comment by j824h** [comment]
Somehow this issue remained open, and now https://github.com/microsoft/vscode/issues/179688 makes this relevant again.

So can we merge those Modern themes here?
What was the procedure, back in 2021 @BlueStaggo?

**Comment by sean-mcmanus** [comment]
@j824h I don't know how https://github.com/microsoft/vscode/issues/179688 is related -- that is a change to the VS Code themes and this issue is about the C/C++ Theme extension and the VS C++ themes. Maybe there are VS Code related theme changes that  impact our theme? Can you provide examples?

**Comment by j824h** [comment]
Oh, I had to look at the title carefully to see that this is about VS C++.
It just has been my inaccurate assumption that the update was to follow the VS Code default theme.

So there is not an official connection that Microsoft makes between the themes for the two products, is that right?
The design can flow back and forth at some points following the trend, maybe that was all.

**Comment by sean-mcmanus** [comment]
Yeah, the design of the C/C++ Theme was intended to match VS and not VS Code. Our team hasn't been tracking the theme changes being made by VS Code. If they've made some theme improvement that you believe the C/C++ theme should react to then we could consider that.
